<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Buscar Exposição</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Buscar Exposição</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filtrosGroup">
     <property name="title">
      <string>Filtros de Busca</string>
     </property>
     <layout class="QFormLayout" name="filtrosLayout">
      <property name="verticalSpacing">
       <number>8</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelNome">
        <property name="text">
         <string>Nome da Exposição:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nome">
        <property name="placeholderText">
         <string>Digite o nome da exposição</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLocal">
        <property name="text">
         <string>Local:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="local">
        <property name="placeholderText">
         <string>Digite o local da exposição</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelDataInicio">
        <property name="text">
         <string>Data (Ano):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="ano">
        <property name="minimum">
         <number>1800</number>
        </property>
        <property name="maximum">
         <number>2100</number>
        </property>
        <property name="value">
         <number>2024</number>
        </property>
        <property name="specialValueText">
         <string>Qualquer</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkAtiva">
        <property name="text">
         <string>Apenas exposições ativas</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="resultadosGroup">
     <property name="title">
      <string>Resultados da Busca</string>
     </property>
     <layout class="QVBoxLayout" name="resultadosLayout">
      <item>
       <widget class="QTableWidget" name="tabelaResultados">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nome</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Data</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Local</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Obras</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="botoesFrame">
     <layout class="QHBoxLayout" name="botoesLayout">
      <item>
       <widget class="QPushButton" name="btnBuscar">
        <property name="text">
         <string>Buscar</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLimpar">
        <property name="text">
         <string>Limpar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnAbrir">
        <property name="text">
         <string>Abrir Exposição</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFechar">
        <property name="text">
         <string>Fechar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
